src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -1716,6 +1716,7 @@ def _get_load_balancer_sku(self, enable_validation: bool = False) -> Union[str,
         load_balancer_sku = safe_lower(self.raw_param.get("load_balancer_sku", CONST_LOAD_BALANCER_SKU_STANDARD))
         # try to read the property value corresponding to the parameter from the `mc` object
         if (
+            load_balancer_sku is None and
```

Comment:
not possible to be None, its default value is `"standard"`

Reply:
removed. Thanks!